lib/intercom_flutter.dart

```diff
@@ -121,7 +121,9 @@ class Intercom {
   }
 
   static Future<dynamic> sendTokenToIntercom(String token) {
-    print("Start sending token to Intercom");
+    if (token != null && token.isNotEmpty) {
+      print("Start sending token to Intercom");
+    }
```

Comment:
This might be a cleaner solution:
```suggestion
    assert(token != null && token.isNotEmpty);
    print("Start sending token to Intercom");
```
What do you think? In native code, null tokens are ignored with this PR, so maybe an assertion will notice developers while debugging it. Assertions are stripped from release builds.

Reply:
Thanks @Zazo032!